cli/search/search.go

```diff
@@ -115,6 +124,49 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitRemote returns commits list from specified remote git repo.
+func GetCommitsFromGitRemote(repo string) ([]string, error) {
+	commits := []string{}
+
+	if _, err := exec.LookPath("git"); err != nil {
+		return nil, fmt.Errorf("'git' is required to search among existing commits")
+	}
+
+	tempRepoPath, err := os.MkdirTemp("", "versions_log")
+
```

Comment:
Most of our code does not insert empty line between an expression and error check right after it.

Reply:
fixed